pkg/vmprovider/providers/vsphere/virtualmachine/configspec_test.go

```diff
@@ -1,24 +1,23 @@
-// Copyright (c) 2022 VMware, Inc. All Rights Reserved.
+// Copyright (c) 2023 VMware, Inc. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
 package virtualmachine_test
 
 import (
 	goctx "context"
 
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
```

Comment:
While you're here, instead of shuffling the import around, you can replace where the sink logger is used here with `suite.GetLogger().WithValues("vmName", vm.GetName()),` like we do in other tests so any log output is shown in the tests.

Reply:
Ah.  Thanks so much for pointing that out.  Fixed.